### PR TITLE
Focus investigation table

### PR DESCRIPTION
**Pull request by kahama94**
fix: #190

### CODE REVIEW

File: src/containers/pages/FocusInvestigation/active/tests/__snapshots__/index.test.tsx.snap [hunk 1]
```diff
@@ -267,319 +271,134 @@ exports[`containers/pages/ActiveFocusInvestigation renders ActiveFocusInvestigat
...
                     "Header": "",
-                    "accessor": "province",
+                    "accessor": "plan_title",
+                    "minWidth": 100,
                   },
                 ],
               },
               Object {
-                "Header": "District",
+                "Header": "FI STATUS",
```
Comment (moshthepitt) [inline src/containers/pages/FocusInvestigation/active/tests/__snapshots__/index.test.tsx.snap]: I dont think that this should be capitalized.

**Comment by kahama94** [comment]
Screenshot from 2019-07-23 13-32-15
Screenshot from 2019-07-23 13-31-55

**Comment by rowo** [comment]
@moshthepitt @kahama94 
- Please revert back to the investigation name as link, instead of having the actions column. There's a little more context on https://github.com/onaio/reveal-frontend/issues/196#issuecomment-514376552. 
- As far as the general table styling with the text wrapping — I think Matt and I are okay with at this point. It is functional.

**Comment by kahama94** [comment]
Removed the action column added link as investigation name.
AwesomeScreenshot-Current-Focus-Investigations-Reveal-2019-07-24-09-07-66

**Comment by moshthepitt** [comment]
@kahama94 here is the latest mockup https://github.com/onaio/reveal-frontend/issues/190#issuecomment-514393012

**Pull request merged**